package.json

```diff
@@ -33,14 +33,17 @@
     "clean": "watchman watch-del-all && rm -rf dist"
   },
   "dependencies": {
+    "hoist-non-react-statics": "2.x.x",
     "lodash": "4.x.x",
-    "mobx": "3.x.x",
```

Comment:
why did you remove mobx from the dependencies?

Reply:
mobx has moved to devDependencies because it's being bundled into a single dist bundle by webpack (should help avoiding isolateGlobalState issues)